src/pages/workspace/tags/WorkspaceTagsSettingsPage.tsx

```diff
@@ -36,9 +36,9 @@ function WorkspaceTagsSettingsPage({route, policyTags}: WorkspaceTagsSettingsPag
     const hasEnabledOptions = OptionsListUtils.hasEnabledOptions(Object.values(policyTags ?? {}).flatMap(({tags}) => Object.values(tags)));
     const updateWorkspaceRequiresTag = useCallback(
         (value: boolean) => {
-            Tag.setPolicyRequiresTag(route.params.policyID, value);
+            Tag.setPolicyRequiresTag(route.params.policyID, value, policyTagLists[0].orderWeight);
```

Comment:
This switch is for the per workspace tag requirement, we shouldn't pass any tag list index

Reply:
yep reverting back to the original API command